### PR TITLE
Add OCDs for Italy

### PR DESCRIPTION
**Pull request by zikowang**
Adding OCDs for Italy's senate and camera.

100 for Senate
- 26 Collegio plurinominale
- 74 Collegio uninominale

196 for Camera
- 49 Collegio plurinominale
- 147 Collegio uninominale

### CODE REVIEW

File: identifiers/country-it.csv [hunk 1]
```diff
@@ -1,2 +1,346 @@
...
+ocd-division/country:it/senato:campania/cp:1,Collegio plurinominale Campania - 01
+ocd-division/country:it/senato:campania/cp:2,Collegio plurinominale Campania - 02
+ocd-division/country:it/senato:campania/cu:1,Collegio uninominale Campania - 01
+ocd-division/country:it/senato:campania/cu:2,Collegio uninominale Campania - 02
+ocd-division/country:it/senato:campania/cu:3,Collegio uninominale Campania - 03
+ocd-division/country:it/senato:campania/cu:4,Collegio uninominale Campania - 04
+ocd-division/country:it/senato:campania/cu:5,Collegio uninominale Campania - 05
+ocd-division/country:it/senato:campania/cu:6,Collegio uninominale Campania - 06
+ocd-division/country:it/senato:campania/cu:7,Collegio uninominale Campania - 07
+ocd-division/country:it/senato:diaspora-_africa_asia_oceania_e_antartide,Abroad - Africa Asia Oceania and Antarctica
```
Comment (HKSenior) [inline identifiers/country-it.csv:243]: The dash can be removed here and below.

```suggestion
ocd-division/country:it/senato:diaspora_africa_asia_oceania_e_antartide,Abroad - Africa Asia Oceania and Antarctica
```

File: identifiers/country-it/senato.csv [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+ocd-division/country:it/senato:sardegna,Circoscrizione Sardegna
+ocd-division/country:it/senato:sicilia,Circoscrizione Sicilia
+ocd-division/country:it/senato:toscana,Circoscrizione Toscana
+ocd-division/country:it/senato:trentino-alto_adige,Circoscrizione Trentino-Alto Adige
+ocd-division/country:it/senato:umbria,Circoscrizione Umbria
+ocd-division/country:it/senato:valle_d_aosta,Circoscrizione Valle D'Aosta
+ocd-division/country:it/senato:veneto,Circoscrizione Veneto
+ocd-division/country:it/senato:diaspora-_europa,Abroad - Europe
+ocd-division/country:it/senato:diaspora-_america_settentrionale_e_centrale,Abroad - North and Central America
+ocd-division/country:it/senato:diaspora-_america_meridionale,Abroad - South America
```
Comment (HKSenior) [inline identifiers/country-it/senato.csv:24]: The dash can be removed here and below.

```suggestion
ocd-division/country:it/senato:diaspora_america_meridionale,Abroad - South America
```

**Review by HKSenior — CHANGES_REQUESTED** [review]

File: identifiers/country-it/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+This folder lists the following:
+
+* Collegio plurinominale and Collegio uninominale of the Senate of Italy
+  - [Regions](https://www.governo.it/it/costituzione-italiana/parte-seconda-ordinamento-della-repubblica/titolo-v-le-regionile-province-e-i#131:~:text=%5B40%5D-,Art.%20131,-%5B41%5D)
+  - [Abroad regions](https://www.esteri.it/it/servizi-consolari-e-visti/italiani-all-estero/votoestero/elezionipolitiche/)
+  - [senato (senate / upper house)](https://dait.interno.gov.it/territorio-e-autonomie-locali/sut/elenco_collegi_senato.php)
+  - Collegio plurinominale as **cp**
+  - Collegio uninominale as **cu**
+* Collegio plurinominale and Collegio uninominale of the Camera of Italy
```
[the remark this answers is not in the document]
Comment (zikowang) [reply identifiers/country-it/README.md]: Good call, added english translation to readme.

File: identifiers/country-it.csv (hunk 1 above)
Comment (zikowang) [reply identifiers/country-it.csv]: adjusted

File: identifiers/country-it/senato.csv (hunk 2 above)
Comment (zikowang) [reply identifiers/country-it/senato.csv]: adjusted

**Comment by zikowang** [comment]
@HKSenior thanks a lot for the review.

The changes requested changes have been implemented.

**Review by HKSenior — APPROVED** [review]

**Comment by jpmckinney** [comment]
Instead of `cameri` and `senato`, how about `lower` and `upper`? That way jurisdictions can be comparable, instead of having local type IDs for lower and upper chambers of parliament.

The readme links to regions, but regions are not the same as Circoscrizione. There can be multiple Circoscrizione per region, for the lower house: https://it.wikipedia.org/wiki/Collegi_elettorali_della_Camera_dei_deputati_del_2020

It also looks like the single-member constituencies should be children of the multi-member constituencies.

Overall, I think the hierarchy would be:

* region
  * upper (circoscrizione) – can maybe skip this level if we assume that the relationship will remain 1-to-1 in future
    * mmc (multi-member constituency)
      * smc or ed (single-member constituency or electoral district)
  * lower (circoscrizione)
    * mmc (multi-member constituency)
      * smc or ed (single-member constituency or electoral district)

**Comment by jpmckinney** [comment]
Looks good!

> There are no multi-member constituencies for the Circoscrizione Valle D'Aosta and Trentino-Alto Adige in [senate](https://dait.interno.gov.it/territorio-e-autonomie-locali/sut/elenco_collegi_senato.php).
And there are no multi-member constituencies for the Circoscrizione Valle D'Aosta in [camrea](https://dait.interno.gov.it/territorio-e-autonomie-locali/sut/elenco_collegi_camera.php).

Aha, for those, I guess `ed` will be a child of `upper`/`lower`.

**Comment by zikowang** [comment]
Alright, I updated the files, adde Regions and adjusted the OCDs for upper and lower house.

I also enhanced the name of the single-member constituencies.

Please have another look.

**Comment by zikowang** [comment]
> This touches country-lt.csv (Lithuania) as well?

Good catch! That was mistakenly pushed -> fixed.

**Comment by sguenther85** [comment]
> * This touches country-lt.csv (Lithuania) as well?
> * Although I had suggested it, now that I see it, I think the `region:` level is repetitive, and also might imply that the mmc and ed are with respect to the regional government, whereas these are about representation at the federal level. Therefore, it might be better to remove the region level. Curious to hear from other maintainers.

Hi, I would also suggest leaving out the region. They can be problematic for several reasons:
- the same thing you said @jpmckinney : might imply that the mmc and ed are with respect to the regional government. That's why the ed in germany are directly under country for the “federal election”
- if constituencies with numbers/names could span or switch regions
- generally no added value, except more work

**Comment by zikowang** [comment]
@jpmckinney should I remove the `region:` level so we can move on?

**Comment by jpmckinney** [comment]
Yes please!

**Comment by zikowang** [comment]
Alright. Regions removed from country IT OCDs.

Please have a look again.

**Review by jpmckinney — APPROVED** [review]

**Review by HKSenior — APPROVED** [review]

**Pull request merged**